File: JS/sumEvenWithErrorBetter.js
function sumEvenNumbers(numbers) {
    var total = 0;
    if (numbers == null) return 0
    for (var i = 0; i < numbers.length; i++) {
        if (numbers[i] % 2 === 0) {
            total += numbers[i];
        }
    }
    return total;
}

var numbers = [1, 2, 3, 4, 5];

console.log(sumEvenNumbers(null));
console.log(sumEvenNumbers(numbers));  // output: 6"

File: Python/sumEvenWithErrorBetter.py
def sumEvenNumbers(numbers):
    if numbers == None:
        return 0
    total = 0
    for i in range(len(numbers)):
        if numbers[i] % 2 == 0:
            total += numbers[i]
    return total

numbers = [1, 2, 3, 4, 5]

print(sumEvenNumbers(None))
print(sumEvenNumbers(numbers)) # output: 6
